packages/toolpad-app/src/runtime/evalJsBindings.ts

```diff
@@ -1,7 +1,15 @@
 import { BindingEvaluationResult, JsRuntime } from '@mui/toolpad-core';
-import { set as setObjectPath } from 'lodash-es';
 import { mapValues } from '@mui/toolpad-utils/collections';
-import { updatePath } from '../utils/immutability';
+// TODO: remove these lodash-es imports
```

Comment:
Why move this function out of `immutability.ts`?

Reply:
It introduces type unsafety, we shouldn't promote using this style of programming